src/vm-repair/azext_vm_repair/custom.py

```diff
@@ -91,13 +91,17 @@ def create(cmd, vm_name, resource_group_name, repair_password=None, repair_usern
             os_image_urn = _fetch_compatible_windows_os_urn(source_vm)
             os_type = 'Windows'
 
+        public_ip_name = '""'
```

Comment:
can you just use "" instead of both ' and "?

Reply:
The documentation says it needs to be "" to avoid the public IP creation on 'az vm create'. Used '""' to ensure I passed the "" characters and not just an empty string.

![image](https://github.com/user-attachments/assets/ba3e8dae-4bef-449f-b682-6498ecb2457e)

https://learn.microsoft.com/en-us/cli/azure/vm?view=azure-cli-latest#az-vm-create

Reply:
I noticed that none of the code passes azdev style even without my modifications. Im working on a 2nd commit for this PR that fixes all the pre-existing errors found my flake8 and pylint